### ISSUE TITLE
Manual edition of groups raise errors

### ISSUE THREAD

**Issue by Jibec** [YunoHost/issues#1493]
Hi team,

the issue: for many apps, adding the app to a group raises an error.

```
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/moulinette/interfaces/api.py", line 439, in process
    ret = self.actionsmap.process(arguments, timeout=30, route=_route)
  File "/usr/lib/python2.7/dist-packages/moulinette/actionsmap.py", line 527, in process
    return func(**arguments)
  File "/usr/lib/moulinette/yunohost/user.py", line 782, in user_permission_update
    sync_perm=sync_perm)
  File "/usr/lib/moulinette/yunohost/log.py", line 287, in func_wrapper
    result = func(*args, **kwargs)
  File "/usr/lib/moulinette/yunohost/permission.py", line 158, in user_permission_update
    new_permission = _update_ldap_group_permission(permission=permission, allowed=new_allowed_groups, sync_perm=sync_perm)
  File "/usr/lib/moulinette/yunohost/permission.py", line 471, in _update_ldap_group_permission
    hook_callback('post_app_addaccess', args=[app, ','.join(effectively_added_users, sub_permission)])
TypeError: join() takes exactly one argument (2 given)
```

Version:
```
admin@ynh:~$ sudo yunohost --version
yunohost: 
  repo: now
  version: 3.7.0.2
yunohost-admin: 
  repo: testing
  version: 3.7.0.1
moulinette: 
  repo: testing
  version: 3.7.0
ssowat: 
  repo: testing
  version: 3.7.0
```

![image](https://user-images.githubusercontent.com/474883/69502052-37983400-0f0b-11ea-8131-ec8cb967be17.png)

(By the way, the green message when adding a new user isn't really useful as username is missing.)

The log file only contains two lines:

> 2019-11-24 22:30:13,820: DEBUG - The permission database has been resynchronized
> 2019-11-24 22:30:16,505: DEBUG - La configuration de SSOwat généré

**Comment by alexAubin** [comment]
It should be fixed in 3.7.0.3 released Saturday

**Comment by Jibec** [comment]
It works indeed fine now. Thank you
